src/index.html

```diff
@@ -15,8 +15,79 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav>
+        <ul>
+          <img
+            class="img"
+            src="/src/images/logo.png"
+            alt="img"
+          />
+          <a
+            class="list"
+            href="Apple"
+          >
+            Apple
+          </a>
+          <a
+            class="list"
+            href="Samsung"
+          >
+            Samsung
+          </a>
+          <a
+            class="list"
+            href="Smartphones"
+          >
+            Smartphones
+          </a>
+          <a
+            class="list"
+            href="data-qa=hover"
+          >
```

Comment:
The `href` property of the `<a>` tag should not contain spaces. If you want to create an anchor link, it should start with the `#` symbol. For example, `href="#data-qa=hover"`. It seems like you wanted to add a custom data attribute. If so, it should be added as a separate attribute, not inside `href`. For example, `data-qa="hover"`.